gst/mqtt/mqttcommon.h

```diff
@@ -21,4 +21,28 @@
 #define GST_MQTT_ELEM_NAME_SINK "mqttsink"
 #define GST_MQTT_ELEM_NAME_SRC "mqttsrc"
 
+#define GST_MQTT_LEN_MSG_HDR    512
+/**
+ * @brief GST_BUFFER_MEM_MAX in gstreamer/gstbuffer.c is 16 so that 15 memory
+ * slots ramained in GstBuffer, which we are going to publish and subscribe.
```

Comment:
Why 15 memory slots? not 16 slots?
If memory allocation scheme is not very trivial, please describe it somewhere.

Reply:
Are you putting metadata of tensor[0] at mqtt header and metadata of tensor[1..15] at message body?

Reply:
@myungjoo The message header consumes one slot of GST_BUFFER_MEM_MAX.

Reply:
> Are you putting metadata of tensor[0] at mqtt header and metadata of tensor[1..15] at message body?

Yes. I did. But that concept had been dumped. I will remove this line.

Reply:
@myungjoo Fixed.